src/nvim/testdir/test_breakindent.vim

```diff
@@ -261,7 +262,7 @@ function Test_breakindent15()
   call s:test_windows('setl breakindent briopt= ts=8 sw=8')
   vert resize 30
   norm! 4a1234567890
-  exe "normal! >>\<C-V>3f0x"
+  exe "normal! >>^\<C-V>3f0x"
```

Comment:
still wondering why we are tweaking random test actions instead of setting the default.